lib/web/fetch/formdata.js

```diff
@@ -154,6 +154,15 @@ class FormData {
       this[kState].push(entry)
     }
   }
+
+  toString () {
```

Comment:
```js
Object.prototype.toString === FormData.prototype.toString
```

we can't add public methods to FormData

Reply:
thankyou ı used your suggesion

Reply:
How can I apply it? 🤔

Reply:
is there a reason that you didn't use inspect.custom?

Reply:
I made an update can you check

Reply:
toString has to be removed entirely

Reply:
Do you have a chance to check? There may be some errors, but I seem to have recovered

Reply:
There is a small error in my test, I tried to solve it, but I encountered an error in the form of FormData.prototype.toString or plain use 

test/fetch/formdata-inspect-custom.js

```
test at test/fetch/formdata-inspect-custom.js:7:1
✖ FormData class custom inspection (1.781208ms)
  AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
  + actual - expected
  
  + FormData {
  +   [Symbol(state)]: [
  +     {
  +       name: 'username',
  +       value: 'john_doe'
  +     },
  +     {
  +       name: 'email',
  +       value: 'john@example.com'
  +     }
  +   ]
  + }
  - 'FormData:\nusername: john_doe\nemail: john@example.com\n'
      at TestContext.<anonymous> (/Users/mert/Desktop/openSource/undici/test/fetch/formdata-inspect-custom.js:14:10)
      at Test.runInAsyncScope (node:async_hooks:206:9)
      at Test.run (node:internal/test_runner/test:631:25)
      at Test.start (node:internal/test_runner/test:542:17)
      at startSubtest (node:internal/test_runner/harness:216:17) {
    generatedMessage: true,
    code: 'ERR_ASSERTION',
    actual: {},
    expected: 'FormData:\nusername: john_doe\nemail: john@example.com\n',
    operator: 'strictEqual'
  }

```

Reply:
const { inspect } = require('node:util')

assert.deepStrictEqual(inspect(formData), expected)

Reply:
Thank you very much for your suggestions 🙏